CHANGELOG.md

```diff
@@ -151,6 +169,18 @@ the APIs in these libraries are stable, and are ready for production use.
   `GOOGLE_CLOUD_CPP_STORAGE_USE_LEGACY_HTTP` to any value to use the legacy
   implementation. We are planning to remove the legacy code and the
   `GOOGLE_CLOUD_CPP_STORAGE_USE_LEGACY_HTTP` environment variable by 2022-12.
+- feat(storage): improve error messages on stalled uploads ([#9744](https://github.com/googleapis/google-cloud-cpp/pull/9744))
+- doc(storage): update broken docs link ([#9738](https://github.com/googleapis/google-cloud-cpp/pull/9738))
+- doc: fix some broken links exposed by the use of mdformat ([#9725](https://github.com/googleapis/google-cloud-cpp/pull/9725))
+- feat(storage): equality for Native IAM types ([#9649](https://github.com/googleapis/google-cloud-cpp/pull/9649))
+- doc(storage): fix comments for Notification requests ([#9642](https://github.com/googleapis/google-cloud-cpp/pull/9642))
+- feat(storage): SA credentials default to self-signed JWTs ([#9629](https://github.com/googleapis/google-cloud-cpp/pull/9629))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- feat: group GUAC `Options` in an `OptionsList` ([#9643](https://github.com/googleapis/google-cloud-cpp/pull/9643))
+- fix(rest): cache credentials ([#9620](https://github.com/googleapis/google-cloud-cpp/pull/9620))
```

Comment:
If this is something `storage `used to do, I would delete the entry.

If not, should we move the entry under `storage`?

Reply:
I will leave it here for now.  Yes, only storage uses these right now, but that will eventually change.